tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -171,6 +171,12 @@ describe('main_html5 wrapper tests', function () {
     expect(vtc.notifyParameters).to.eql([vtc.interface.EVENTS.PLAYING]);
   });
 
+  it('should notify ASSET_DIMENSION on first \'canPlay\' event', function(){
+    vtc.interface.EVENTS.ASSET_DIMENSION = "assetDimension";
+    $(element).triggerHandler("canplay");
+    expect(vtc.notifyParameters).to.eql([vtc.interface.EVENTS.ASSET_DIMENSION, {width: 0, height: 0}]);
```

Comment:
if the video object used in testing doesn't handle manual setting i see it may not be possible